Resources/config/services.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+
+        <!-- commands -->
+        <service id="doctrine.fixtures_load_command" class="Doctrine\Bundle\FixturesBundle\Command\LoadDataFixturesDoctrineCommand">
```

Comment:
should be private

Reply:
Actually no, as we need to to support older versions. Hence getting the command from the container directly rather than from the command loader.

Reply:
Hmm, NVM. I forgot a public alias was created in such case, so should be okay.

Reply:
Reverted. See:

https://github.com/symfony/symfony/blob/2661da415e108a479589bb943d4c12c8088c6e38/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php#L31-L33

I knew [I already hit](https://travis-ci.org/lexik/LexikJWTAuthenticationBundle/jobs/290768849#L555) the issue somewhere 😉